lib/info.js

```diff
@@ -53,6 +53,14 @@ module.exports = function getInfo(link, options, callback) {
 
   // Try getting config from the video page first.
   var url = VIDEO_URL + id;
+  
+  // Add Language Support
+  if (options.lang) {
+    url = url + "&hl=" + options.lang;
+  } else {
+    url = url + "&hl=en";
+  }
+  
```

Comment:
Would rather put this with the one line above and use the `||` operator. The quotes used should also be singlequotes since that is what the project uses.

`'&hl'=(options.lang || 'en)`